services/server/tests/handlers/report.test.ts

```diff
@@ -0,0 +1,161 @@
+import db, { faker } from "@fixtures/db";
+import handlers from "@/handlers/report";
+import { IReport, IUser } from "@litespace/types";
+import { mockApi } from "@fixtures/mockApi";
+import { forbidden, notfound } from "@/lib/error";
+import { expect } from "chai";
+import { reports } from "@litespace/models";
+import { first } from "lodash";
+import { nameof } from "@litespace/utils";
+
+const find = mockApi<
+  IReport.FindApiPayload,
+  object,
+  object,
+  IReport.FindApiResponse
+>(handlers.find);
+const update = mockApi<IReport.UpdateApiPayload>(handlers.update);
+const create = mockApi<IReport.CreateApiPayload>(handlers.create);
+
+describe(nameof(find), () => {
```

Comment:
The first outer can be just `Reports` and then each handler should has its own describe block.